pkg/server/errorpage/templates.go

```diff
@@ -9,7 +9,12 @@ var defaultErrorPageTemplate = template.Must(template.New("defaultErrorPageTempl
 const defaultErrorPageTemplateString = `<!DOCTYPE html>
 <html lang="en-us" data-test-id="login">
   <head>
-    <title>Error - OKD</title>
+    <title>
+
+        {{ .Locale.Error }}
+
+       · OKD
+    </title>
```

Comment:
@rhamilto `Jeckyll` added newlines within `<title>` is that ok? should they be removed?

Reply:
It is fine.  The whitespace is ignored by the browser.